### PR TITLE
fix colored border and highlights

### PR DESCRIPTION
**Pull request by swus13**
add style from default css for borders and colored highlights

### CODE REVIEW

File: templates/munstrap/static/css/style-munstrap.css [hunk 1]
```diff
@@ -35,10 +35,34 @@ ul.groupview, ul.groupview ul {
 	list-style-type: none;
 }
 .munin-icon {
-   background: url(../img/logo-munin.png) left top; 
+   background: url(../img/logo-munin.png) left top;
```
Comment (TafThorne) [inline templates/munstrap/static/css/style-munstrap.css]: A previous.  I prefer layout changes in a seperate patch posted after the functional ones. It is entirely up to the actual maintainers of this project to say what they prefer though.

**Comment by sumpfralle** [comment]
@swus13: do you want to integrate @TafThorne comment's into your pull request?

However: it seems like no one raised concerns, thus I would merge your changes.

It is a pity that there are so few people involved with layout and web design in munin. I am sorry for the lack of response regarding your improvements.

**Comment by swus13** [comment]
sorry, appending white spaces removed the IDE by itself, which is not my intention to change that.
I hope this is no problem now and can remain so in the commit.

**Comment by sumpfralle** [comment]
yes, these few added formatting changes are no problem - thank you!

**Pull request merged**